doc/mgr/rgw.rst

```diff
@@ -37,6 +37,25 @@ system user has no more access keys, it is removed.
 
 Commands
 --------
+::
+
+  ceph rgw apply -i spec.yaml
+
+Create a new realm + zonegroup + zone and deploy rgw daemons via the
+orchestrator using the information specified in the YAML file. In case
+the realm, zonegroup or zone already exist the module will not re-create
+them again, instead it will use their information. Following is an example
```

Comment:
"instead it will use their information" for what purpose? I think if we're going to mention that the info from the existent realm/zonegroup/zone is getting used somewhere else we should also say for what.

Reply:
text has changed